deeds-dapp-webapp/src/main/webapp/vue-app/dapp/layout/components/topbar/LanguageSelector.vue

```diff
@@ -65,7 +67,92 @@ export default {
   }),
   methods: {
     changeLanguage(lang) {
+      const pageName = window.location.pathname.split('/')[2];
+      let uri = '';
+      if (this.language === 'fr') {
+        switch (pageName) {
+        case 'place-de-marche': 
+          uri = 'marketplace';
+          break;
+        case 'portefeuille': 
+          uri = 'portfolio';
+          break;
+        case 'visite-guidee': 
+          uri = 'tour';
+          break;
+        case 'livre-blanc': 
+          uri = 'whitepaper';
+          break;
+        case 'tokenomics-fr': 
+          uri = 'tokenomics';
+          break;
+        case 'qui-sommes-nous': 
+          uri = 'about-us';
+          break;
+        case 'deeds-fr': 
+          uri = 'deeds';
+          break;
+        case 'mentions-legales': 
+          uri = 'legals';
+          break;
+        case 'rejoindre-dao': 
+          uri = 'stake';
+          break;
+        case 'proprietaires': 
+          uri = 'owners';
+          break;
+        case 'farm-fr': 
+          uri = 'farm';
+          break;
+        case 'locataires': 
+          uri = 'tenants';
+          break;
+        default: uri = '';
+        }
+      } else {
+        switch (pageName) {
+        case 'marketplace': 
+          uri = 'place-de-marche';
+          break;
+        case 'portfolio': 
+          uri = 'portefeuille';
+          break;
+        case 'tour': 
+          uri = 'visite-guidee';
+          break;
+        case 'whitepaper': 
+          uri = 'livre-blanc';
+          break;
+        case 'tokenomics': 
+          uri = 'tokenomics-fr';
+          break;
+        case 'about-us': 
+          uri = 'qui-sommes-nous';
+          break;
+        case 'deeds': 
+          uri = 'deeds-fr';
+          break;
+        case 'legals': 
+          uri = 'mentions-legales';
+          break;
+        case 'stake': 
+          uri = 'rejoindre-dao';
+          break;
+        case 'owners': 
+          uri = 'proprietaires';
+          break;
+        case 'farm': 
+          uri = 'farm-fr';
+          break;
+        case 'tenants': 
+          uri = 'locataires';
+          break;
+        default: uri = '';
+        }
+      }
       this.$store.commit('selectLanguage', lang);
+      window.history.replaceState('', '', `${this.parentLocation}/${this.$t(uri)}`); 
+      window.location.reload();     
```

Comment:
Shouldn't reload page IMO & pages correspondance should be added inside Map (JSON Object key => value)

```suggestion
```